pkg/flow/packet_record.go

```diff
@@ -0,0 +1,40 @@
+package flow
+
+import (
+	"encoding/binary"
+	"io"
+)
+
+//var plog = logrus.WithField("component", "packet/PacketRecord")
+
+type RawByte byte
+
+type PacketRecord struct {
+	Stream []byte
+}
+
+// NewPacketRecord contains packet bytes
+func NewPacketRecord(
+	stream []byte,
+	len uint32,
+) *PacketRecord {
+	pr := PacketRecord{}
+	pr.Stream = make([]byte, len)
+	pr.Stream = stream
+	return &pr
+}
+
+// ReadPacketFrom reads a PacketRecord from a binary source, in LittleEndian order
+func ReadRawPacketFrom(reader io.Reader) (*PacketRecord, error) {
```

Comment:
Can you rename this to what it does `ReadRawPacketFromBuffer` or just `ReadRawPacket`